### PR TITLE
feat: Add planet types to CrawlPlanets

### PR DESCRIPTION
**Pull request by phated**
This updates the CrawlPlanets plugin to be able to crawl specific planet types. It also fixes the planet level max (not that it really matters because you aren't crawling lvl 8-9).

I got rid of the ability to crawl all planets, but could see it being added back in another PR.

### CODE REVIEW

File: content/productivity/crawl-planets/plugin.js [hunk 1]
```diff
@@ -129,8 +171,9 @@ function capturePlanets(fromId, minCaptureLevel, maxDistributeEnergyPercent) {
   const candidates_ = df.getPlanetsInRange(fromId, maxDistributeEnergyPercent)
     .filter(p => (
       p.owner !== df.account &&
-      p.owner === "0x0000000000000000000000000000000000000000" &&
-      p.planetLevel >= minCaptureLevel
+      players.includes(p.owner) &&
+      p.planetLevel >= minCaptureLevel &&
+      p.planetType === planetType
```
Comment (PawlowskiAdrian) [inline content/productivity/crawl-planets/plugin.js:176]: In future we can modify selection to list select few from list. Example: selection = add element like in repeat attack plugin

File: content/productivity/crawl-planets/plugin.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+const players = [
+  "0x0000000000000000000000000000000000000000",
+];
+
+const typeNames = Object.keys(planetTypes);
 
 class Plugin {
   constructor() {
+    this.planetType = 1;
     this.minPlanetLevel = 3;
```
Comment (PawlowskiAdrian) [inline content/productivity/crawl-planets/plugin.js:20]: Let's add layer to allow more flexibility during crawl.

```suggestion
this.minPlanetLevelTo = 3;
this.minPlanetLevelFrom = 0;
this.maxPlanetLevelFrom = 3;
```

File: content/productivity/crawl-planets/plugin.js [hunk 3]
```diff
@@ -107,6 +147,8 @@ class Plugin {
     container.appendChild(percent);
     container.appendChild(levelLabel);
     container.appendChild(level);
```
Comment (PawlowskiAdrian) [inline content/productivity/crawl-planets/plugin.js:149]:
```suggestion
container.appendChild(levelLabelFrom);
container.appendChild(levelFrom);
container.appendChild(levelLabelToMin);
container.appendChild(levelToMin);
container.appendChild(levelLabelToMax);
container.appendChild(levelToMax);
```

**Comment by phated** [comment]
I'm going to merge this as-is but happy to review/merge @PawlowskiAdrian's changes on top of it!

**Pull request merged**